src/calc.py

```diff
@@ -1,12 +1,13 @@
-def add(a, b):
+def add(a, b, c = 0):
```

Comment:
I know we already were using single-letter var names, but we can do better. I suggest the following
```suggestion
def add(a, b, third_operand = 0):
```